applications/FluidDynamicsApplication/custom_elements/symbolic_stokes.h

```diff
@@ -0,0 +1,405 @@
+    virtual ~SymbolicStokes();
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    ///@}
+    ///@name Operations
+    ///@{
+
+    /// Create a new element of this type
+    /**
+    * Returns a pointer to a new SymbolicStokes element, created using given input.
+    * @param NewId the ID of the new element
+    * @param ThisNodes the nodes of the new element
+    * @param pProperties the properties assigned to the new element
+    * @return a Pointer to the new element
+    */
+    Element::Pointer Create(IndexType NewId,
+        NodesArrayType const& ThisNodes,
+        Properties::Pointer pProperties) const override;
+
+    /// Create a new element of this type using given geometry
+    /**
+    * Returns a pointer to a new FluidElement element, created using given input.
+    * @param NewId the ID of the new element
+    * @param pGeom a pointer to the geomerty to be used to create the element
+    * @param pProperties the properties assigned to the new element
+    * @return a Pointer to the new element
+    */
+    Element::Pointer Create(IndexType NewId,
+        GeometryType::Pointer pGeom,
+        Properties::Pointer pProperties) const override;
+
+    /// Computes the elemental LHS and RHS elemental contributions
+    /**
+     * Given a distance function, computes the time integrated Left Hand Side (LHS)
+     * and Right Hand Side elemental contributions for the two-fluid element.
+     * @param rLeftHandSideMatrix elemental stiffness matrix
+     * @param rRightHandSideVector elemental residual vector
+     * @param rCurrentProcessInfo reference to the current process info
+     */
+    void CalculateLocalSystem(
+        MatrixType &rLeftHandSideMatrix,
+        VectorType &rRightHandSideVector,
+        ProcessInfo &rCurrentProcessInfo) override;
+
+    /// Computes the elemental RHS elemental contribution
+    /**
+     * Given a distance function, computes the time integrated Right Hand Side (RHS)
+     * elemental contribution for the two-fluid element.
+     * @param rRightHandSideVector elemental residual vector
+     * @param rCurrentProcessInfo reference to the current process info
+     */
+    void CalculateRightHandSide(
+        VectorType &rRightHandSideVector,
+        ProcessInfo &rCurrentProcessInfo) override;
+
+
+    /**
+     * @brief MassMatrix Calculate the local mass matrix.
+     * @param rFluidStress Viscous stress in the fluid given in Voigt notation
+     * @param rCurrentProcessInfo Current ProcessInfo values (input)
+     */
+    void Calculate( const Variable<Vector>& rVariable,
+                    Vector& rOutput,
+                    const ProcessInfo& rCurrentProcessInfo) override;
+
+    /// Auxiliar element check function
+    /**
+     * This function calls the base element check method and adds the
+     * current element check implementations
+     * @param rCurrentProcessInfo reference to the current process info
+     */
+    int Check(const ProcessInfo &rCurrentProcessInfo) override;
```

Comment:
```suggestion
    int Check(const ProcessInfo &rCurrentProcessInfo) const override;
```

Reply:
Remove the Check. Now using the base class Check, which has been changed to const.